utils/tests.go

```diff
@@ -74,6 +74,13 @@ func GenerateResource() meroxa.Resource {
 	return GenerateResourceWithNameAndStatus("", "")
 }
 
+func GenerateFlinkJob() meroxa.FlinkJob {
```

Comment:
I see, I was already wondering how you can mock out a flink job in those unit tests :eyes: